src/pages/ReportView/index.tsx

```diff
@@ -66,7 +57,7 @@ export class ReportView extends React.Component<Props, State> {
         let action = !this.props.loading && report ? report.id : '';
```

Comment:
Another case of `let` where you probably want `const` instead. Just in case you accidentally reassign these variables, the language can catch you there.

Reply:
Thanks, Mike. Change has done :)